src/Core/Grid/Definition/Factory/Security/Session/CustomerGridDefinitionFactory.php

```diff
@@ -98,6 +98,27 @@ protected function getName(): string
      */
     protected function getColumns(): ColumnCollectionInterface
     {
+        $deleteAction = $this->buildDeleteAction(
```

Comment:
Instead of doing all this manually maybe we could improve `buildDeleteAction` and add an optional `$options` parameter to override the default values in the method?

Since you needed this for at least four use cases in this PR it's probably worth it

Reply:
It's done.